spec/abilities/admin_set_ability_spec.rb

```diff
@@ -1,178 +1,374 @@
 # frozen_string_literal: true
 require 'cancan/matchers'
 
-RSpec.describe Hyrax::Ability do
+RSpec.describe Hyrax::Ability, :clean_repo do
```

Comment:
General comments that apply throughout the test:
* some comments about solr_document abilities were on the wrong lines
* unless noted, all tests are unchanged
* tests for AdminSets were duplicated to test AdministrativeTests